features/testbot/summary.feature

```diff
@@ -0,0 +1,77 @@
+@routing @basic @testbot
+Feature: Basic Routing
+
+    Background:
+        Given the profile "testbot"
+
+    @smallest
+    Scenario: Checking 
+        Given the node map
+            | a | b |  | c | d | e |
+
+        And the ways
+            | nodes |
+            | ab    |
+            | bc    |
+            | cd    |
+            | de    |
+
+        When I route I should get
+            | from | to | route          | summary  |
+            | a    | e  | ab,bc,cd,de,de | ab, bc   |
+            | e    | a  | de,cd,bc,ab,ab | de, bc   |
+            | a    | b  | ab,ab          | ab       |
+            | b    | d  | bc,cd,cd       | bc, cd   |
+
+    @smallest
+    Scenario: Check handling empty values
+        Given the node map
+            | a | b |  | c |   | d | f |
+            |   |   |  |   |   | e |
+
+        And the ways
+            | nodes | name |
+            | ab    | ab   |
+            | bc    | bc   |
+            | cd    |      |
+            | de    | de   |
+            | df    | df   |
+
+        When I route I should get
+            | from | to | route          | summary  |
+            | e    | a  | de,,bc,ab   | de, bc   |
```

Comment:
Well, this is not really a bug, at least not in post processing. It is how to announce name-changes with no-name.

I added https://github.com/Project-OSRM/osrm-backend/pull/2367 to handle this situation a bit better.

In addition, the string needs to be `de,,bc,ab,ab` (currently ,,->bc is not announced, will change https://github.com/Project-OSRM/osrm-backend/pull/2367)